python/apps/taiga/src/taiga/users/api/__init__.py

```diff
@@ -178,6 +183,37 @@ async def update_my_user(request: Request, form: UpdateUserValidator) -> User:
     )
 
 
+#####################################################################
+# delete user
+#####################################################################
+
+
+# TODO
+
+
+#####################################################################
+# delete info user
+#####################################################################
+
+
+@routes.my.get(
+    "/my/user/delete-info",
+    name="my.user.delete-info",
+    summary="Get user delete info",
+    response_model=UserDeleteInfoSerializer,
+    responses=ERROR_401,
+)
+async def get_user_delete_info(request: Request) -> UserDeleteInfoSerializer:
+    """
+    Get the delete info about a user.
+    """
+    if request.user.is_anonymous:
+        # NOTE: We force a 401 instead of using the permissions system (which would return a 403)
+        raise ex.AuthorizationError("User is anonymous")
```

Comment:
Couldn't we code this permission validation in the usual way with a new PermissionComponent which implement  the `is_authorized` method?

```
class IsNotAnonymous(PermissionComponent):
    async def is_anonymous(self, user: "AnyUser", obj: Any = None) -> bool:
        return bool(user and not user.is_anonymous)
```
```
    await check_permissions(permissions=LIST_DELETE_INFO, user=request.user, obj=user)
```
Take the code just as an idea to get what I'm trying to say